integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/LandingPageSpecification.groovy

```diff
@@ -24,7 +24,6 @@ class LandingPageSpecification extends GebReportingSpec {
   def setup() {
     db.clearDb()
     elite2Api.stubAgencyDetails('LPI')
-    elite2Api.stubAssessments('B2345YZ')
   }
```

Comment:
moving stubAssessments() back into the individual tests wasnt actually needed in the end but i left it in as it more accurately emulates scenarios where there is no existing cat